Remotion/Validation/Globalization/LocalizedValidationMessageFactory.cs

```diff
@@ -52,7 +52,7 @@ public enum ResourceIdentifier
       ValueMustMatchPredicateValidationMessage,
       ValueMustMatchRegularExpressionValidationMessage,
       ValueMustNotBeEmptyCollectionValidationMessage,
-      ValueMustNotBeEmptyStringValidationMessage,
+      ValueMustNotBeEmptyValueValidationMessage,
```

Comment:
Please split this up for byte[] and string. We have several cases (see null values) where we have the same message with different values but still used typed resources. That way, we can decide this on a case-by-case basis and don't have to rewrite the IDs just to customate the message for a specific type. it also allows types-specific overrides in project code so having types-specific messages is also a possible extension point / feature provided by the infrastrucuture.